templates/myself.conf.j2

```diff
@@ -1,7 +1,7 @@
 # {{ ansible_managed }}
 Client {
   Name = {{ item.name }}
-  Maximum Concurrent Jobs = 20
+  Maximum Concurrent Jobs = {{ bareos_client_maximum_concurrent_jobs | default('20') }}
```

Comment:
This may forbid to define different `Maximum Concurrent Jobs` for different clients.

Reply:
Fixed